### ISSUE TITLE
Screen FW boots to splash animation like no card is there

### ISSUE THREAD

**Issue by TTVert** [malebuffy/Kobra-Max-Cobra-Kai-UI-and-Firmware#7]
I am at a loss here.  I have tried 4 different SD cards ranging from 8-32GB.  All formatted FAT32/4096 (Although I did try NTFS, FAT and exFAT as a test).  I have tried 2 screens.  I only have DWIN_SET and its contents in the roof of the SD card.  I have it  inserted into the screen slot and ensured it is clicked securely in place.  I am completely out of ideas.  I would love if one of you who has successfully flashed this on the kobra Max would let me know the model and revision # of their screen to see if it differs from mine.   I am out of ideas at this point. 
![image](https://github.com/malebuffy/Kobra-Max-Cobra-Kai-UI-and-Firmware/assets/24374538/9c07b18d-6fab-4b8e-bb68-553042353d0e)
Thanks
Dave

**Comment by malebuffy** [comment]
I am on vacation 2000km away from my printer...hope someone else can chime in.

However I used the SD card that came with the printer, 8gb formated as Fat32 @+and 4096 sectors. Also Check that the DWIN_SET folder doesn't contain another DWIN_SET Folder (Happens sometimes when you unzip)

**Comment by TTVert** [comment]
> 

So the issue seems to be the fact the screen is a TJC screen.  I need to find FW that supports that screen, NOT DWIN.  Even speaking w/ AC they are clueless and they should know what screens they are shipping out w/ products.

**Comment by malebuffy** [comment]
I don't think that there is custom firmware atm that supports that screen.

**Comment by TTVert** [comment]
> I don't think that there is custom firmware atm that supports that screen.

Yeah i'm starting to find that to be the case :(  Very sad.  Do you know "IF" I found a kobra max DWIN screen I would be ok or are there other changes mainboard side?   I would assume there wouldn't be but what do I know?

**Comment by malebuffy** [comment]
I believe there wouldn't be a problem as the screen is kind of separate from the mainboard. I am not aware of a completely new mainboard for the max. I know they start to use the USB C type Kobra 2 mainboard now, but they ship it also as a replacement to people with the Dwin screen so it shouldn't be a problem.

That beeing said, I was also not aware they ship Max's with other type of screens...

**Comment by TTVert** [comment]
> I believe there wouldn't be a problem as the screen is kind of separate from the mainboard. I am not aware of a completely new mainboard for the max. I know they start to use the USB C type Kobra 2 mainboard now, but they ship it also as a replacement to people with the Dwin screen so it shouldn't be a problem.
> 
> That beeing said, I was also not aware they ship Max's with other type of screens...

Well I know many devs have made DWIN icon packs for the kobra max (Including AC themselves) so there have to be  a least 2 screens now.  Even ACs instructions mention the DWIN_SET folder and nothing about these TJC screens that I have.  1 came w/ the printer and AC sent me a replacement which is also a TJC screen.

**Comment by jcst40** [comment]
This is the DWIN screen if you need a replacement:

[LCD touchscreen](https://www.aliexpress.com/item/1005003290517252.html?fbclid=IwAR3Hxza8p_aFFHeVTxs4OrE8gJcK88c1kcQb7V7P23cSeinsHWqD4Ky0yaA)

(you need to solder XH2.54-6P straight pin connector) 

What you claim make sense as The kobra 2 firmware includes 2 different UI firmwares file types.
I think Anycubic is just trying to get less open source with their products. They uploaded a non functional source for the K2 and as they release new products completely leave behind upgrades for the previous printers even if they just as old as 1 year old.

**Comment by TTVert** [comment]
> This is the DWIN screen if you need a replacement:
> 
> [LCD touchscreen](https://www.aliexpress.com/item/1005003290517252.html?fbclid=IwAR3Hxza8p_aFFHeVTxs4OrE8gJcK88c1kcQb7V7P23cSeinsHWqD4Ky0yaA)
> 
> (you need to solder XH2.54-6P straight pin connector)
> 
> What you claim make sense as The kobra 2 firmware includes 2 different UI firmwares file types. I think Anycubic is just trying to get less open source with their products. They uploaded a non functional source for the K2 and as they release new products completely leave behind upgrades for the previous printers even if they just as old as 1 year old.

Thank you for that!  Do you know if it's a capacitive or resistive display?

Dave
